mvnxparser/MVNXStreamReader/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright: (C) 2017 iCub Facility
+# Author: Diego Ferigo
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+# Load dependencies
+find_package(Qt5Xml REQUIRED)
+
+# Build the libraries
+add_library(MVNXStreamReader MVNXStreamReader.cpp)
```

Comment:
I never saw any consistency in library names.. 
just to see some example from my codyco-superbuild : 
`YARP_OS`, `gazebo_yarp_camera`, `idyntree-core`, `idyntree-core`, `skinDynLib`, `wholeBodyDynamics_IDLServer`. I am agnostic to the style used in the library names.